src/datasets/utils/resources/tasks.json

```diff
@@ -4,9 +4,9 @@
         "subtasks": [
             "keyword-spotting",
             "speaker-identification",
-            "audio-intent-classification",
-            "audio-emotion-recognition",
-            "audio-language-identification"
+            "speaker-intent-classification",
+            "emotion-recognition",
+            "speaker-language-identification"
```

Comment:
i think those should prob be updated in `huggingface_hub` @osanseviero

Reply:
Done in https://github.com/huggingface/hub-docs/pull/95/commits/18c70dac6bb83cfa56d5e7594f5ca45cefbe7049